v1.8.1/default/theme.json

```diff
@@ -0,0 +1,22 @@
+{
+    "name": "default_v1_8_1",
```

Comment:
are we going to use this name instead of `default`?

Reply:
we thought it would be easier for the customer to just download and upgrade instead of downloading, unzipping, renaming it, zipping again and uploading. I can change it back to default if you think it's better

Reply:
the idea of this repo is to be a mirror of what we ship on the packages, but we can have add-ons, like you can upload a zip file with that update you proposed, and a readme file to explain why there's that file in the repo, what's the difference between the original one